src/compiler/LessCompiler.ts

```diff
@@ -113,6 +119,25 @@ export class LessCompiler implements ICompiler {
         })
     }
 
+    /**
+     * Checks improper LESS escaping
+     * @param cssOutput CSS string output
+     * @private
+     */
+    private checkImproperLessEscaping(cssOutput: string): string | undefined {
+        const cssOutputArray = cssOutput.split('\n');
+        const foundImproperLessEscaping = cssOutputArray.reduce((acc: string[], val: string) => {
```

Comment:
This is making warnings for all `calc` expressions, whether they are escaped or not. The css output will not have the escaping included I believe.

Reply:
True. I am on it. It was my bad assumption. Thanks for the hint 👍